src/storage-operators/src/persist_source.rs

```diff
@@ -233,10 +227,15 @@ where
         desc_transformer,
         Arc::new(metadata.relation_desc),
         Arc::new(UnitSchema),
-        move |stats| {
+        move |stats, frontier| {
+            let time_range = if let Some(lower) = frontier.as_option().copied() {
```

Comment:
In practice the frontier is always a single timestamp, but I thought we might prefer to not bake that assumption in. That means that the else branch never really shows up in practice...